CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
 # Git LFS Changelog
 
+## 1.3.0 (21 July 2016)
+
+### Features
+
+* use proxy from git config  #1358 (@technoweenie)
+* Enhanced transfers part 5: external custom transfer adapters #1367 (@sinbad)
+* In git 2.9+, run "git lfs pull" in submodules after "git lfs clone" #1373 (@sinbad)
+* cmd,doc,test: teach `git lfs track --{no-touch,verbose,dry-run}` #1344 (@ttaylorr)
+* ⏳ Retry transfers with expired actions #1350 (@ttaylorr)
+* Safe track patterns #1346 (@ttaylorr)
+* Enhanced transfers part 4: resuming uploads via tus.io protocol #1303 (@sinbad)
+* Add checkout --unstaged flag #1262 (@orivej)
+* cmd/clone: add include/exclude via flags and config #1321 (@ttaylorr)
+* Improve progress reporting when files skipped #1296 (@sinbad)
+* Lock commands #1256 (@ttaylorr)
+* Enhanced transfers: part 3 (resumable downloads) #1297 (@sinbad)
+* Enhanced transfers: part 2 #1279 (@sinbad)
+* Enhanced transfers: part 1 #1265 (@sinbad)
+* Implement support for GIT_SSH_COMMAND #1260 (@pdf)
+* api: add Force option to unlock requests #1259 (@ttaylorr)
+* API Client and Lock Service #1236 (@ttaylorr)
```

Comment:
Combined the lock commands together in https://github.com/github/git-lfs/pull/1388/commits/409b5cad7823eed0fd18d0c3e04259f2ad187346

As for the transfer adapters, I'm going to tag in @sinbad to see what he feels is best 😄